src/en/news/blog/2023/reddit-is-10k-IOPs-achieable-with-NVMes/index.md

```diff
@@ -0,0 +1,147 @@
+
+# Acknowledgements
+
+First, thank you to [Clyso](https://www.clyso.com) for funding this work to benefit the Ceph community.  Thank you as well to [IBM](https://ibm.com)/[Red Hat](https://redhat.com) and [Samsung](https://samsung.com) for providing the upstream Ceph community with the hardware used for this testing.  Thank you to all of the Ceph developers that have worked tirelessly to make Ceph great.  Finally, thank you to [/u/nNaz](https://www.reddit.com/user/nNaz/) for reaching out and driving some new Ceph benchmarking!
+
+# User Setup
+
+The user indicated in the original reddit post for this article that they have 6 nodes that each have 2 4TB Seagate FireCuda NVMe drives and 64GB of memory.  They didn't mentioned what kind of CPUs or networking they have, but both are potentially important for this test.  Conveniently, they already had an fio invocation in mind that they wanted to evaluate:
+
+```fio --randrepeat=1 --ioengine=libaio --direct=1 --gtod_reduce=1 --name=test --filename=test --bs=4k --iodepth=64 --size=150G --readwrite=randrw --rwmixread=75``` 
+
+While we don't have any FireCuda drives in the upstream lab, we do have nodes that have multiple 4TB Samsung PM983 drives and 128GB of memory each.
+
+# Cluster Setup 
+
+Nodes | 10 x Dell PowerEdge R6515
+-- | --
+CPU | 1 x AMD EPYC 7742 64C/128T
+Memory | 128GiB DDR4
+Network | 1 x 100GbE Mellanox ConnectX-6
+NVMe | 6 x 4TB Samsung PM983
+OS Version | CentOS Stream release 8
+Ceph Version 1 | Pacific v16.2.13 (built from source)
+Ceph Version 2 | Quincy v17.2.6 (built from source)
+Ceph Version 3 | Reef v18.1.0 (built from source)
+<br>
+
+To match the user's setup as closely as possible, we only used 6 of the nodes in the cluster with 2 OSDs running on a single NVMe drive each for Ceph. The remaining nodes were used as client nodes.  All nodes are located on the same Juniper QFX5200 switch and connected with a single 100GbE QSFP28 link.  Ceph was deployed and fio tests were launched using [CBT](https://github.com/ceph/cbt/).  An important OS level optimization on Intel systems is setting the tuned profile to either "latency-performance" or "network-latency".  This primarily helps by avoiding latency spikes associated with CPU C/P state transitions.  AMD Rome based systems do not appear to be as sensitive in this regard, and I have not confirmed that tuned is actually restricting C/P state transitions, however the tuned profile was nevertheless set to "network-latency" for these tests.
```

Comment:
By co-incidence today on a Xeon Silver 4114 system I used `turbostat` to verify that both these profiles limited C-state transitions to no deeper than C1.

TuneD